File: eSalon/eSalon.Model/Ocjena.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eSalon.Model
{
    public class Ocjena
    {
        public int OcjenaId { get; set; }

        public int UslugaId { get; set; }

        public int KorisnikId { get; set; }

        public int Vrijednost { get; set; }

        public DateTime DatumOcjenjivanja { get; set; }
    }
}

File: eSalon/UI/esalon_mobile/lib/models/ocjena.dart
import 'package:json_annotation/json_annotation.dart';

part 'ocjena.g.dart';

@JsonSerializable()
class Ocjena {
  int? ocjenaId;
  int? uslugaId;
  int? korisnikId;
  int? vrijednost;
  DateTime? datumOcjenjivanja;

  Ocjena(
    this.ocjenaId,
    this.uslugaId,
    this.korisnikId,
    this.vrijednost,
    this.datumOcjenjivanja,
  );

  factory Ocjena.fromJson(Map<String, dynamic> json) => _$OcjenaFromJson(json);

  Map<String, dynamic> toJson() => _$OcjenaToJson(this);
}
